logdetective/utils.py

```diff
@@ -199,3 +202,15 @@ def validate_url(url: str) -> bool:
     if not (result.path or result.netloc):
         return False
     return True
+
+
+def load_prompts(path: str | None) -> PromptConfig:
```

Comment:
I'd like to ask you to create at least one test case for this function.

Reply:
Thanks for reminding me. Tests incoming.